src/curry/once.ts

```diff
@@ -0,0 +1,55 @@
+const onceResult = '__once__'
```

Comment:
We can use `Symbol` to avoid any potential collisions (but I don't expect too much of them :smile:  )

Reply:
I did that originally, but since we use isolated declarations, we would have to export the symbol :)

Reply:
Okay, I found a way to use a symbol without exporting it! I'm also using that same symbol to represent an uncalled state, instead of using the `in` and `delete` operators.